bodhi/client/bindings.py

```diff
@@ -29,12 +29,15 @@
 import datetime
 import functools
 import getpass
+import itertools
 import json
 import logging
 import os
 import re
 import textwrap
 
+from textwrap import wrap
```

Comment:
Since this is part of the stdlib, it should be grouped with the other stdlib imports. However, ```textwrap``` is already imported above, so I recommend using ```textwrap.wrap()``` instead of just ```wrap()```. If you prefer to import it this way, please group the import with the other stdlib imports for PEP-8.